app/src/main/java/org/literacyapp/service/synchronization/AuthenticationJobService.java

```diff
@@ -41,4 +62,36 @@ private boolean rescheduleIfTrainingJobServiceIsRunning(){
         }
         return isFaceRecognitionTrainingJobServiceRunning;
     }
+
+    private boolean isScreenTurnedOff(){
+        boolean isScreenTurnedOff = false;
+        DisplayManager displayManager = (DisplayManager) getApplicationContext().getSystemService(Context.DISPLAY_SERVICE);
+        for (Display display : displayManager.getDisplays()) {
+            if (display.getState() != Display.STATE_ON) {
+                isScreenTurnedOff = true;
+            }
+        }
+        Log.i(getClass().getName(), "isScreenTurnedOff: " + isScreenTurnedOff);
+        return isScreenTurnedOff;
+    }
+
+    private boolean didTheMinimumTimePassSinceLastAuthentication(){
+        boolean didTheMinimumTimePassSinceLastAuthentication = true;
+        LiteracyApplication literacyApplication = (LiteracyApplication) getApplicationContext();
+        DaoSession daoSession = literacyApplication.getDaoSession();
+        AuthenticationEventDao authenticationEventDao = daoSession.getAuthenticationEventDao();
+        // Get only the last AuthenticationEvent
+        List<AuthenticationEvent> authenticationEvents = authenticationEventDao.queryBuilder().orderDesc(AuthenticationEventDao.Properties.Time).limit(1).list();
+        if (authenticationEvents.size() > 0){
+            AuthenticationEvent authenticationEvent = authenticationEvents.get(0);
+            long lastAuthenticationTime = authenticationEvent.getTime().getTime().getTime();
```

Comment:
Can we log these 3 long properties as a Date object to make it more human readable in the log output?